packages/cli/test/integration-1.test.ts

```diff
@@ -235,68 +234,55 @@ test('[vc build] should build project with corepack and select npm@8.1.0', async
     expect(contents).toEqual(['home', 'shim']);
   } finally {
     delete process.env.ENABLE_EXPERIMENTAL_COREPACK;
-    delete process.env.SKIP_YARN_COREPACK_CHECK;
   }
 });
 
 test('[vc build] should build project with corepack and select pnpm@7.1.0', async () => {
-  try {
-    process.env.ENABLE_EXPERIMENTAL_COREPACK = '1';
-    process.env.SKIP_YARN_COREPACK_CHECK = '1';
-    const directory = await setupE2EFixture('vc-build-corepack-pnpm');
-    const before = await exec(directory, 'pnpm', ['--version']);
-    const output = await execCli(binaryPath, ['build'], { cwd: directory });
-    expect(output.exitCode, formatOutput(output)).toBe(0);
-    expect(output.stderr).toMatch(/Build Completed/gm);
-    const after = await exec(directory, 'pnpm', ['--version']);
-    // Ensure global pnpm didn't change
-    expect(before.stdout).toBe(after.stdout);
-    // Ensure version is correct
-    expect(
-      await fs.readFile(
-        path.join(directory, '.vercel/output/static/index.txt'),
-        'utf8'
-      )
-    ).toBe('7.1.0\n');
-    // Ensure corepack will be cached
-    const contents = fs.readdirSync(
-      path.join(directory, '.vercel/cache/corepack')
-    );
-    expect(contents).toEqual(['home', 'shim']);
-  } finally {
-    delete process.env.ENABLE_EXPERIMENTAL_COREPACK;
-    delete process.env.SKIP_YARN_COREPACK_CHECK;
-  }
+  process.env.ENABLE_EXPERIMENTAL_COREPACK = '1';
```

Comment:
Let's retain the try/finally block here and in the following test (like in the test above)